idp.go

```diff
@@ -157,6 +178,14 @@ func (idp *IdentityProvider) AddAttribute(name string, value string, format stri
 	})
 }
 
+func (idp *IdentityProvider) AddContactPerson(service ContactPerson) {
+	idp.ContactPerson = append(idp.ContactPerson, service)
```

Comment:
@sccalabr `idp.ContactPerson` is unresolved reference, IdentityProvider struct doesn't have ContactPerson as its fields